library/core/src/fmt/mod.rs

```diff
@@ -2097,6 +2097,7 @@ impl<'a> Formatter<'a> {
 }
 
 #[stable(since = "1.2.0", feature = "formatter_write")]
+#[doc(notable_trait)]
 impl Write for Formatter<'_> {
```

Comment:
Hmm, if we decide to land this, I wonder if we should rename `notable_trait` to just `notable`...

Reply:
That would make sense, since it'll cover notable_trait and notable_impl into one

Reply:
If it were to simply be called `notable`, I wonder if there would be usefulness in extending it to other areas too... notable fields? Is that useful? I don't know, but maybe it's worth exploring...